app/screens/AddChildInfoSetup.tsx

```diff
@@ -329,18 +329,25 @@ const AddChildInfoSetup = ({ route, navigation }: Props): any => {
   }
 
 
-  const AddChild = async (isDefaultChild: boolean,isDefaultName: boolean): Promise<any> => {
+  const AddChild = async (isDefaultChild: boolean, isDefaultName: boolean): Promise<any> => {
     await userRealmCommon.getData<ChildEntity>(ChildEntitySchema);
-    let defaultName ;
-    if(isDefaultName){
-       defaultName='Baby';
+    let defaultName;
+    if (isDefaultName) {
+      defaultName = 'Baby';
+    } else {
+      defaultName = name;
+    }
+    let insertData:any = null;
+    if(isDefaultChild){
+      insertData = await getNewChild('', "true", isExpected, plannedTermDate, isPremature, birthDate, defaultName, '', gender, null);
     }else{
-        defaultName= name;
+      insertData = await getNewChild('', "false", isExpected, plannedTermDate, isPremature, birthDate, defaultName, '', gender, null);
     }
-    const insertData: any = await getNewChild('', isExpected, plannedTermDate, isPremature, birthDate, defaultName, '', gender, null);

```

Comment:
If isDefaultChild is a boolean can we use below code?

`insertData = await getNewChild('', isDefaultChild, isExpected, plannedTermDate, isPremature, birthDate, defaultName, '', gender, null);`